tests/common/src/tools/time_test.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gmock/gmock.h>
+
+#include "../log.hpp"
+#include "aos/common/tools/time.hpp"
+
+using namespace testing;
+using namespace aos;
+using namespace aos::time;
+
+class TimeTest : public Test {
+private:
+    void SetUp() override
+    {
+        Log::SetCallback([](LogModule module, LogLevel level, const String& message) {
+            static std::mutex sLogMutex;
+
+            std::lock_guard<std::mutex> lock(sLogMutex);
+
+            std::cout << level.ToString().CStr() << " | " << module.ToString().CStr() << " | " << message.CStr()
+                      << std::endl;
+        });
+    }
+};
+
+TEST_F(TimeTest, Add4Years)
+{
+    /**
+     * Current Time API doesn't provide a possibility to get date/time component(implementation doesn't require that).
```

Comment:
You could check it by Unixnano method to verify that time is advanced by the expected amount of nanoseconds.
TODO: is forbidden to merge. Please remove it. I

Reply:
yes, thank you. done that way.